usr/src/cmd/mdb/common/modules/nsmb/nsmb.c

```diff
@@ -20,9 +20,10 @@
  */
```

Comment:
My review of the mdb stuff will be cursory at best.

Reply:
That's fine.  It had review internally, and we've used this in support situations.